DEVX-1645:  Naming stateful operations (#314)
confluentinc/kafka-tutorials: _includes/tutorials/naming-changelog-repartition-topics/kstreams/markup/dev/run-consumer.adoc

////
  This is a sample content file for how to include a console consumer to the tutorial, probably a good idea so the end user can watch the results
  of the tutorial.  Change the text as needed.

////

> Now go ahead stop the console-producer with a `CTRL+C` and start the console-consumer.  Take note you're running the consumer with the `--from-beggining` option so you'll get all messages sent to the `output` topic.

Reviewer: ```suggestion
Stop the console-producer with a `CTRL+C`, and start the console-consumer.  Take note you're running the consumer with the `--from-beginning` option so you'll get all messages sent to the `output` topic.
```

+++++
<pre class="snippet"><code class="shell">{% include_raw tutorials/naming-changelog-repartition-topics/kstreams/code/tutorial-steps/dev/console-consumer.sh %}</code></pre>
+++++

From your first run you should see the following output:

+++++
<pre class="snippet"><code class="shell">{% include_raw tutorials/naming-changelog-repartition-topics/kstreams/code/tutorial-steps/dev/expected-output.txt %}</code></pre>
+++++

> You should take note that even though this is the output of an aggregation operation, this tutorial configured the streams application to use `StreamsConfig.CACHE_MAX_BYTES_BUFFERING_CONFIG = 0`, so you'll see _**every**_ update from `count()` operation.

Reviewer: ```suggestion
Note that even though this is the output of an aggregation operation, this tutorial configured the streams application to use `StreamsConfig.CACHE_MAX_BYTES_BUFFERING_CONFIG = 0`, so you'll see _**every**_ update from the `count()` operation.
```

> Since the streams application takes the first character to use as the key, so the output of 1-1, 1-2, and 1-3 (key-count) is expected.

Reviewer: ```suggestion
Since the streams application takes the first character to use as the key, the output of `1-1`, `1-2`, and `1-3` (key-count) is expected.
```
